.github/workflows/release.yml

```diff
@@ -14,42 +16,29 @@ jobs:
     if: github.repository == 'd3fc/d3fc'
     needs: build-test-lint
     runs-on: ubuntu-latest
-    env:
-      NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+    permissions:
+      contents: write
+      pull-requests: write
     steps:
       - name: "Checkout repository"
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          ssh-key: ${{secrets.DEPLOY_KEY}}
-
-      - name: "Setup SSH"
-        uses: webfactory/ssh-agent@v0.7.0
-        with:
-          ssh-private-key: ${{ secrets.DEPLOY_KEY }}
 
       - name: "Setup Node"
         uses: actions/setup-node@v4
         with:
           node-version: 20.15.0
           registry-url: https://registry.npmjs.org/
-      
-      - name: "Git config"
-        run: |
-          git config --global user.name 'github-actions[bot]'
-          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
-          git remote set-url origin git@github.com:${{github.repository}}.git
 
       - name: "Install dependencies"
         run: npm ci
 
-      - name: "Running build"
-        run: npm run build
```

Comment:
Maybe a silly question but do we not still need to do this build step as part of this? Can't see anywhere else it's done, so wondered if you figured out we don't need it for the publish?

Reply:
Good spot! My bad